### PR TITLE
Bulk delete for certificate (missing) and pagination for cohort students

### PR DESCRIPTION
**Pull request by alexscoelho**
Bulk delete for certificate (missing) and pagination for cohort students

### CODE REVIEW

File: breathecode/certificate/views.py [hunk 1]
```diff
@@ -135,6 +136,39 @@ def get(self, request, academy_id=None):
...
+        if lookups:
+            items = UserSpecialty.objects.filter(
+                **lookups, academy__id=academy_id)
+
+            for item in items:
+                item.delete()
+
+            return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+        if cohort_id is None or user_id is None:
```
Comment (alesanchezr) [inline breathecode/certificate/views.py:160]: Borra toda la funcionalidad de cohort_id y user_id porque no es parte de bulk.

File: breathecode/certificate/views.py [hunk 2]
```diff
@@ -103,7 +103,8 @@ def post(self, request, cohort_id, academy_id=None):
         all_certs = []
 
         if cohort_users.count() == 0:
-            raise APIException("There are no users with STUDENT role in this cohort")
+            raise APIException(
```
Comment (jefer94) [inline breathecode/certificate/views.py:106]: Change it for ValidationException and pass a keywork param called slug

**Review by jefer94 — CHANGES_REQUESTED** [review]

**Comment by jefer94** [comment]
@alexscoelho currently this branch need a pull from development

**Comment by alexscoelho** [comment]
features already in last PR

**Pull request closed without merging**